android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -1209,6 +1214,64 @@ public void annotationsCouldNotBeAdded(String s) {
         }
     };
 
+    private ToolManager.PdfDocModificationListener mPdfDocModificationListener = new ToolManager.PdfDocModificationListener() {
+        @Override
+        public void onBookmarkModified() {
+            WritableMap params = Arguments.createMap();
+            params.putString(ON_BOOKMARK_CHANGED, ON_BOOKMARK_CHANGED);
+            String bookmarkJson = null;
+            if (getPdfDoc() != null) {
```

Comment:
null check should include the whole block, if no doc, the event should not be triggered